README.md

```diff
@@ -6,6 +6,10 @@ JRD is batch application and JRD batch is scheduled with Kubernetes which runs o
 JRD consume data files from an external source, transform that data into the destination format 
 and load the data into JRD database. 
 
+JRD Uses Data-Ingestion-Library (https://github.com/hmcts/data-ingestion-lib) and configure camel routes with
+data-ingestion-lib & JRD custom configuration files & crates camel integration framework which reads CSV files & Stores
```

Comment:
you can mention the sample gradle entry of lib ,will be easy for any new person to implement

Reply:
done.